make/agents/app/build.mk

```diff
@@ -15,7 +15,7 @@ run: fmt vet ## Run a controller from your host.
 # (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
-docker-build: test ## Build docker image with the manager.
```

Comment:
nice, as part of this change I think we can also remove [Go Setup from Push to ghcr workflow](https://github.com/primaza/primaza/blob/da77304464cc2d48d350723121e63ff8ea09fa1d/.github/workflows/push-ghcr.yaml#L21)

Reply:
Rebased and added.